ci/static/ci/css/base.css

```diff
@@ -20,6 +20,7 @@
   font-family: monospace;
   max-height: 50em;
   overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
```

Comment:
Are you sure you want the dash out front like that?

Reply:
Every site I've been to... thats how its done.

Apple-[Supported CSS Properties](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariCSSRef/Articles/StandardCSSProperties.html)

[developer-mozilla](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-overflow-scrolling)

CSS tricks.com: https://css-tricks.com/snippets/css/momentum-scrolling-on-ios-overflow-elements/

Yeah, after what Derek said "you're not looking hard enough" I made sure that is the tag.

Reply:
Hmm - ok. That appears to be correct after a bit of Googling. I don't fully understand CSS, does anyone?